tools/reconciler/Makefile

```diff
@@ -0,0 +1,15 @@
+NAME = reconciler
```

Comment:
What do you think about making the Reconciler Team CODEOWNERS for this new directory?

Reply:
Done